1632/01-gen/08.txt

```diff
@@ -1,22 +1,22 @@
-Y záwoniáł Pán wonnośći wdźięcżney / y rzekł Pán w ſercu ſwem : Nie będę więcey przeklináł źiemi dla cżłowieká ; ábowiem myśl ſercá cżłowiecżego złá jeſt od młodośći jego / nie zátrácę więcey wƺyſtkiego co żyje / jákom teraz ucżynił.
-A póki źiemiá trwáć będźie / śiew y żniwo / y źimno / y gorąco / y láto / y źimá / y dźień / y noc nie uſtáną.
+Y wſpomniał Bóg ná Noego y ná wƺyſtkie zwierzętá / y ná wƺyſtko bydło / które <i>było</i> znim w Korabiu : y przywiódł Bóg wiátr ná źiemię á záſtánowiły śię wody.
+Y záwárte ſą źrzódłá przepáśći / y okná niebieſkie : y záhámowány jeſt deƺcz z niebá.
+Y wróćiły śię wody zwirzchu źiemie idąc y wracájąc śię : y opádły wody po ſkońcżeniu ſtá y piąćidźieśiąt dni.
+Y odpocżynął Korab mieśiącá śiódmego / śiedmnaſtego dniá <i>tegoż</i> mieśiącá / ná górách Arárád.
+A wody zćiekáły y opadáły / áż do dźieśiątego mieśiącá : Dźieśiątego <i>bowiem</i> mieśiącá / pierwƺego dniá okazáły śię wierzchy gór.
+Y ſtáło śię po ſkońcżeniu czterdźieſtu dni / otworzył Noe / okno Korabiá / które był uczynił.
+Y wypuśćił kruká : który tám y ſám latájąc / záśię śię wracał / áż oſchły wody ná źiemi.
+Potym wypuśćił gołębicę od śiebie : áby obaczył jeśli opádły wody z wirzchu źiemie.
+Ale nienálazƺy gołębicá odpoczynieniá ſtopie nogi ſwojey wróćiłá śię do niego do Korabiá : jeƺcze bowiem wody <i>były</i> po wƺyſtkiey źiemi : Y wyćiągnąwƺy / rękę ſwoję wźiął ją / y wniósł ją do śiebie do Korabiá.
+A poczekawƺy jeƺcze drugie śiedm dni / powtóre wypuśćił gołębicę z Korabiá.
+Y wróćiłá śię do niego gołębicá pod wiecżór : á oto różdżká oliwy urwána wuśćiech jey / á ták poznał Noe że opádły wody zwierzchu źiemie.
+Y cżekał jeƺcze drugie śiedm dni / y wypuśćił gołębicę : która śię więcey niewróćiłá do niego.
+Y ſtáło śię ƺeść ſetnego y pierwƺego roku / mieśiącá pierwƺego / <i>dniá</i> pierwƺego / oſchły wody z źiemie : y zjął Noe przykryćie Korabiá / á ujrzał że oſechł wierzch źiemie.
+A mieśiącá wtórego / dwudźieſtego śiódmego dniá <i>tegoż</i> mieśiącá / oſchłá źiemiá.
+Y rzekł Bóg do Noego : mówiąc ;
+Wynidź z Korabiá / ty y żoná twojá / y Synowie twoji y żony ſynów twojich z tobą.
+Wƺyſtkie zwierzętá które <i>ſą</i> z tobą / z wƺelkiego ćiáłá / z ptáſtwá y z bydłá / y z wƺelkiej gádźiny płażájącey śię po źiemi / wywiedź z ſobą á niech śię rozpładzáją ná źiemi / y niech roſną / y rozmnażáją śię ná źiemi.
+Y wyƺedł Noe / y ſynowie jego / y żoná jego / y żony ſynów jego znim.
+Wƺelkie zwierzę / wƺelka gádźiná / y wƺelkie ptáſtwo / wƺyſtko co śię płaza po źiemi według rodzájów ſwoijch / wyƺły z Korabiá.
+Zátym zbudował Noe ołtarz PAnu / y wźiął z káżdego bydła czyſtego / y z káżdego ptáſtwá czyſtego / y ofiárował cáłopalenia ná ołtarzu onym.
+Y záwoniał PAn wonnośći w dźięczney / y rzekł PAn w ſercu ſwym : Nie będę więcey przeklinał źiemie dla człowieká : ábowiem miśl ſercá człowiecżego zła <i>jeſt</i> od młodośći jego / nie zátrácę więcey wƺyſtkiego co żyje / jákom <i>teraz</i> uczynił.

```

Comment:
Jako że we wszystkich innych miejscach jest „wdźięczney”, mniemam, iż i tu winno być tak samo.